services/gundeck/test/integration/API.hs

```diff
@@ -136,9 +138,18 @@ removeUser g c _ s = do
     deleteUser g user
     ntfs <- listNotifications user Nothing g
     liftIO $ do
-        tokens <- Cql.runClient s (Push.lookup user Push.Quorum)
+        logger <- Log.new Log.defSettings
```

Comment:
ah.  i'm pretty sure we already have a logger somewhere nearby.  this works, but i'll see if i can improve on it.